Dockerfile.vertex

```diff
@@ -0,0 +1,38 @@
+FROM python:3.7.9-buster
+WORKDIR /root
+
+# ----------------------------------------
+# Apt-get deps
+# ----------------------------------------
+RUN apt-get -qqy update && \
```

Comment:
I'll follow up to see what we can trim here, I suspect Damien grabbed this from a working example we have to make fast progress. If possible it'd be great to not have to introduce another dockerfile

Reply:
We can remove this

Reply:
Sure, this was included as a minimal public domain dockerfile that worked since the default prefect image wasn't working. Was that resolved?